core/src/main/java/com/adobe/aem/dot/common/util/PathUtil.java

```diff
@@ -85,15 +85,27 @@ public static String stripLastPathElement(String path) {
   }
 
   /**
-   * Return the final folder or filename.
+   * For a Windows' path, strip off the drive (ie "C:") from the path.  Assumes only one colon in the path, which is
+   * part of the drive (i.e. "C:\...")
    * @param path A path
-   * @return path Path without the final folder. If not a path (no slashes), returns original string.
+   * @return path Path without drive.
    */
-  public static String getLastPathElement(String path) {
-    if (path.contains(":")) {
+  public static String stripOffDrive(String path) {
```

Comment:
The name of this method threw me off initially. Perhaps `removeDriveLetterPrefix` would be a bit more clear?